tuplex/core/src/DataSet.cc

```diff
@@ -163,6 +164,47 @@ namespace tuplex {
         return *op->getDataSet();
     }
 
+    DataSet &DataSet::sort(std::vector<std::size_t> order, std::vector<std::size_t> orderEnums) {
+        // if error dataset, return itself
+        if (isError())
+            return *this;
+
+        assert(_context);
+        assert(this->_operator);
+
+        // parameter checking
+        assert(order.size() == orderEnums.size());
```

Comment:
maybe do this check also in release mode, i.e. explicit if with logger().error()... I fear users might pass wrongly params. Maybe there's a slightly more robust version imaginable? I.e., indices for which orderEnum is not specified?